datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/snmp_tile/{check_name}/README.md

```diff
@@ -0,0 +1,41 @@
+# Agent Check: snmp_{integration_name}
+
+## Overview
+
+This check monitors [{integration_name}][1].
+For details of monitored metrics see the [Network Device Monitoring][2] documentation.
+
+## Setup
+
+### Installation
+
+{install_info}
+
+### Configuration
+
+1. <List of steps to setup this Integration>
+
+### Validation
+
+<Steps to validate integration is functioning as expected>
+
+## Data Collected
+
+### Metrics
+
+The {integration_name} integration does not include any metrics.
```

Comment:
> The {integration_name} integration does not include any metrics.

I think all snmp vendor tiles will have metrics, so, this ^ might be a bit misleading.

Reply:
This should be `For details of monitored metrics see the [SNMP integration tile][2]`